### PR TITLE
Update access and permissions

### PR DESCRIPTION
**Pull request by jeppester**
We are now using google groups for creating team specific accounts for github and heroku.
These guidelines need to also reflect the.

One thing I'm not entirely certain about is heroku ownership.
1. Should the team account be the owner of heroku projects (payment details need to be set up)?
2. Or should we use the admin@abtion.com heroku account as owner, and just add the team account as admin?

Suggestions are welcome :-)

Update: The heroku team owns the project and has payment details.

### CODE REVIEW

File: tools_and_services/access_and_permissions.md [hunk 1]
```diff
@@ -33,21 +33,29 @@
...
+- If it's needed a google group can be set up for the client, e.g. `danish-shipping@abtion.com`.
+- Client groups can be used for creating client specific accounts for github, azure etc.
+
 ## Github
 
 - Access is controlled through teams. Abtioneers are part of teams. Teams have access to repos. E.g. Wordpress team will have access to several repos.
 - To give a better overview, members should not get granted permissions directly to any repo. 
 - The Abtion team has read access to all the apps. All Abtioneers are members of the Abtion team.
 - Every new client with a team will have their own team. E.g. `danish-shipping`
+- For rails projects a client specific github account will be created - using the team's google group. This account must have admin-access the repository.
```
Comment (hcarreras) [inline tools_and_services/access_and_permissions.md:46]: Why do we mention Rails?

File: tools_and_services/access_and_permissions.md [hunk 2]
```diff
@@ -33,21 +33,29 @@
 
 You can request access to vaults through Slack #access-and-permissions.
 
+## Google groups
+- If it's needed a google group can be set up for the client, e.g. `danish-shipping@abtion.com`.
```
Comment (hcarreras) [inline tools_and_services/access_and_permissions.md:37]: We should set it as a standard. 
```suggestion
- A google group will be set up per client, e.g. `danish-shipping@abtion.com`.
```

File: tools_and_services/access_and_permissions.md [hunk 3]
```diff
@@ -33,21 +33,29 @@
...
+- Client groups can be used for creating client specific accounts for github, azure etc.
```
Comment (hcarreras) [inline tools_and_services/access_and_permissions.md:38]:
```suggestion
- The group will be used for creating client-specific accounts for services such as github, azure, etc.
```

**Review by hcarreras — COMMENTED** [review]
Super good to have this documentation revisited. Just wondering about the Rails mention

File: tools_and_services/access_and_permissions.md (hunk 1 above)
Comment (jeppester) [reply tools_and_services/access_and_permissions.md:46]: Probably not needed

**Review by hcarreras — APPROVED** [review]
Awesome

**Pull request merged**